src/sagemaker/mxnet/README.rst

```diff
@@ -4,9 +4,9 @@ Using MXNet with the SageMaker Python SDK
 
 With the SageMaker Python SDK, you can train and host MXNet models on Amazon SageMaker.
 
-Supported versions of MXNet: ``1.3.0``, ``1.2.1``, ``1.1.0``, ``1.0.0``, ``0.12.1``.
+Supported versions of MXNet: ``1.4.0``, ``1.3.0``, ``1.2.1``, ``1.1.0``, ``1.0.0``, ``0.12.1``.
 
-Supported versions of MXNet for Elastic Inference: ``1.3.0``.
+Supported versions of MXNet for Elastic Inference: ``1.3.0``, ``1.4.0``.
```

Comment:
I'd keep the order here consistent with the others (i.e. 1.4 first)

Reply:
oh wow nice catch